discord_slash/context.py

```diff
@@ -193,14 +193,18 @@ async def send(self,
         initial_message = False
         if not self.responded:
             initial_message = True
-            if files:
-                raise error.IncorrectFormat("You cannot send files in the initial response!")
+            if files and not self.defered:
+                await self.defer(hidden=hidden)
+            if self.defered:
+                if self._defered_hidden != hidden:
             if self.deferred:
                 if self._deferred_hidden != hidden:
```

Comment:
That is a small mistake I guess?

Reply:
Seems like I introduced it while merging in new changes :frowning: